jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -22,7 +22,7 @@
         respawn="true" output="screen">
     <rosparam>
       duration: 180
-      charge_level_threshold: 40.0
+      charge_level_threshold: 80.0
```

Comment:
I just found this change has already merged in https://github.com/jsk-ros-pkg/jsk_robot/pull/1431 ( d2e919bead1bbfc6b7d3507aa554773b63c8cda9)